commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/tsv/deserialization/stream/FileListTsvStreamDeserializer.kt

```diff
@@ -15,7 +15,7 @@ import java.io.OutputStream
 internal class FileListTsvStreamDeserializer {
     fun serializeFileList(files: Sequence<BioFile>, fileList: OutputStream) {
         val writer = fileList.bufferedWriter()
-        writeHeaderNames(files.first(), writer)
+        processFirstFile(files.first(), writer)
```

Comment:
Something does not look correct, if you notice we are writing two times now the attributes values of the first file. Once in line 35 and another in line 19. That probablythe reason unit test are failing.

Reply:
I think the problem was coming from the `.first()` method taking out the first element from the sequence. Then, when it starts iterating, it'll miss it since it's already processed